yt/analysis_modules/halo_finding/rockstar/rockstar.py

```diff
@@ -325,7 +325,9 @@ def run(self, block_ratio = 1, callbacks = None, restart = False):
             self.ts._pre_outputs = self.ts._pre_outputs[restart_num:]
         else:
             restart_num = 0
-        self.handler.setup_rockstar(self.server_address, self.port,
+        self.handler.setup_rockstar(
+                    self.server_address.encode('ascii'),
```

Comment:
I think if you use `six.b(self.server_address)` it's going to work for both py2 and py3. Similarly for all the other strings